### ISSUE TITLE
TwitchUpdateCustomRewardBody creates every field

### ISSUE THREAD

**Issue by wolbee** [kanimaru/twitcher#38]
When using `TwitchUpdateCustomRewardBody.from_json()`, every field gets created.

However, with `TwitchService.api.update_custom_reward()`, you only want to supply the fields you want to update - having an empty `title` field or a 0 for `cost` breaks the API call.

**Comment by kanimaru** [comment]
Wich version are you using? Technically I should have fixed that with 30c723d3f1b61f290cad8b5a9194fa6fbecd9b45 just to be sure.

**Comment by wolbee** [comment]
Yeah, I was using the 1.2 release, not the most up-to-date code, my bad.
